### ISSUE TITLE
ePub on Kindle

### ISSUE THREAD

**Issue by ngocvantran** [javierbyte/docs2epub#1]
Sorry for nitpicking but your image (docs/og.jpg) shows a Kindle device which does not support ePub. As a Kindle owner, it made me thought Kindle now support ePub, which unfortunately not.

**Comment by javierbyte** [comment]
@7Pass you are right! I'm using Calibre to read epubs on my kindle, and choose .epub as the default format for the project because I believe it is more popular.

Let me see if I can add .mobi to the downloads :) or find another free ereader mockup.
